System/System.cpp

```diff
@@ -46,13 +47,13 @@ namespace RTE {
 		if (s_CaseSensitive) {
 			if (s_WorkingTree.empty()) {
 				for (const std::filesystem::directory_entry &directoryEntry : std::filesystem::recursive_directory_iterator(s_WorkingDirectory, std::filesystem::directory_options::follow_directory_symlink)) {
-					s_WorkingTree.emplace_back(std::hash<std::string>()(directoryEntry.path().generic_string().substr(s_WorkingDirectory.length())));
+					s_WorkingTree.emplace_back(boost::hash<std::string>()(directoryEntry.path().generic_string().substr(s_WorkingDirectory.length())));
```

Comment:
Why use boost::hash?

Reply:
That fixes implementation differences of std::hash that prevent crossplatform multiplayer from working

Reply:
Migh be worth adding a comment to that effect